tailwind.config.cjs

```diff
@@ -59,6 +59,8 @@ module.exports = {
         white: "#d8e7e7",
         eagle: "#a4b3b3",
         "pop-blue": "#2541ea",
+        'lb-sky': '#6d98cc',
+        'lb-purple': '#8a64e5',
```

Comment:
Oops forgot to reply to this one, the values themselves are consistent with it, however the naming scheme wasn't, I've since read the relevant sections of the branding guideline and addressed the required changes in https://github.com/LadybirdBrowser/ladybird.org/pull/115/commits/9d0589a9210dddbc42a905432c2bdda51ec40dba.